modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -573,6 +572,35 @@ struct iOSAudioIODevice::Pimpl      : public AudioPlayHead,
 
         return session.inputGain == gain;
     }
+    
+    void selectBackMic ()
+    {
+        auto session = [AVAudioSession sharedInstance];
+
+        NSArray *portDescriptions = session.availableInputs;
+        AVAudioSessionPortDescription* builtInMicPort = nil;
+
+        for (AVAudioSessionPortDescription* port in portDescriptions) {
+            if ([port.portType isEqualToString:AVAudioSessionPortBuiltInMic]) {
+                builtInMicPort = port;
+                break;
+            }
+        }
+
+        if (builtInMicPort) {
+            for (AVAudioSessionDataSourceDescription* source in builtInMicPort.dataSources) {
+                if ([source.orientation isEqual:AVAudioSessionOrientationBack]) {
+                    break;
+                }
+            }
+            
+            for (AVAudioSessionDataSourceDescription* source in builtInMicPort.dataSources) {

```

Comment:
Should this but be in a function called `selectCardiodPolarPattern()` ?

Reply:
Sure, I can separate that